certificate-secured-gateway-api/manifests/iam/ExternalDNS_permissions.json

```diff
@@ -0,0 +1,23 @@
+{

```

Comment:
Kann man das hier nicht direkt auch als Terraform machen, wenn du das sowieso schon nutzt?

Reply:
Die Dateien in diesem Repo gehören zu meinem Blogpost wo diese Policy via AWS CLI deployed wird.